app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -43,7 +43,7 @@ class CollectionsController < Hyrax::My::CollectionsController
       # The search builder to find the collections' members
       self.membership_service_class = Collections::CollectionMemberSearchService
 
-      load_and_authorize_resource except: [:index, :create], instance_name: :collection
+      load_and_authorize_resource except: [:index, :create, :new], instance_name: :collection
```

Comment:
Requited to accomodate Hyrax::PcdmCollection instead of ::Collection.

Reply:
can you say more about this? why not load `Hyrax::PcdmCollection` with CanCan's tooling as well?